### ISSUE TITLE
boxed values from observable maps

### ISSUE THREAD

**Issue by mrjjwright** [danielearwicker/boxm#1]
Hi,  I need expando observable properties and would like to create dynamic boxed objects for getting and setting their values.  Using an underlying `observable.map` from mobx, do you think this is possible as this lib was built for regular object properties?

Thanks so much for this lib!

**Comment by danielearwicker** [comment]
You're welcome! 😎 

The aim of `boxm` is to take an object and a property name in a way that can be statically type checked, so it sounds like something different. But maybe if the types of the properties are uniform, i.e. it's `{ [propName: string]: number }` because the dynamic properties are all numbers something could be done.

Can you post a code snippet showing what you're doing?

**Comment by mrjjwright** [comment]
Actually, I refactored my code to use regular properties and it's working super nicely at the moment.  Not sure if I will need this now.  This is a really awesome technique and when I wrote the "stringly" typed stuff to set values back on a model I always wondered if there was a better way.  I especially like that it abstracts out a pattern I was using where I always had to keep track of a target object to mutate.  And it fits in with mobx and TypeScript so nicely.  I am really excited about this one.  I am building a fairly complicated editor and this really helps.  Love your blog!
